app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,7 @@
     <meta name="csrf-token" content="<%= form_authenticity_token %>">
     
```

Comment:
why you leave blank line here

Reply:
it was left by mistake, I will remove it